wallet-new/test/unit/Wallet/Inductive/Cardano.hs

```diff
@@ -230,16 +230,15 @@ equivalentT activeWallet esk = \mkWallet w ->
           let dsl = f inductiveCtxtWallet
           translated <- toCardano ctxt fld dsl
 
-          unless (translated == kernel) $
-            throwError EquivalenceViolation {
-                equivalenceViolationName     = fld
-              , equivalenceViolationEvents   = inductiveCtxtEvents
-              , equivalenceViolationEvidence = NotEquivalent {
+          unless (translated == kernel) .
+            throwError $ EquivalenceViolation
```

Comment:
Your indentation here (2 spaces) is a little inconsistent with how [you've handled similar situations](https://github.com/input-output-hk/cardano-sl/pull/3251/files#diff-ac76adacf26f94ef9feb605ba14e9e89R284) throughout the PR.